packages/contracts/contracts/core/DAO.sol

```diff
@@ -90,13 +93,15 @@ contract DAO is
     function initialize(
         bytes calldata _metadata,
         address _initialOwner,
-        address _trustedForwarder
+        address _trustedForwarder,
+        string calldata daoURI_
```

Comment:
Why is `_metadata` a `bytes calldata` and `daoURI_` a `string`? 
Why not `_daoURI` instead of `daoURI_`?

Reply:
`daoURI_` has to be a `string` by definition https://eips.ethereum.org/EIPS/eip-4824
because `_daoURI` is internally used for the variable and therefore we cannot use that name anymore

Reply:
Would it make any sense for us to have the metadata as `string`?
Just asking, can be eventually dealt with in a separate PR

Reply:
It makes no difference. Both types have a length attribute and can be as long as we want. I would keep it as bytes this way a frontend like etherscan won't try to display it with UTF8